app/concepts/cinema_halls/use_cases/create_seats.rb

```diff
@@ -0,0 +1,46 @@
+module CinemaHalls
+  module UseCases
+    class CreateSeats
+      attr_reader :params
+
+      def initialize(params:)
+        @params = params
+      end
+
+      def call
+        columns = to_range(@params[:columns])
```

Comment:
Why you convert `rows` and `columns` arrays to ranges?

Reply:
It might be not practical but I want to create a request for creating a hall like this:
```
{
  "cinema_hall": {
    "name": "hall 1",
    "capacity": 50,
    "rows": ["A", "F"],
    "columns": [1, 10],
    "not_available": ["A1", "A4", "A5", "A6", "B4", "B5", "B6", "C4", "C5", "C6"]
  }
}
```
I know `rows` wouldn't start with "B" or "K" but I wanted to give a opportunity to describe the first row and also column and play a little bit with Ruby

Reply:
Ok, now I understand 👍